harness/determined/pytorch/_lr_scheduler.py

```diff
@@ -67,8 +65,10 @@ def __init__(
             frequency:
                 Sets the frequency at which the batch and epoch step modes get triggered.
         """
-        check.check_not_none(scheduler)
-        check.check_isinstance(step_mode, LRScheduler.StepMode)
+        if scheduler is None:
+            raise ValueError("scheduler must not be None.")
```

Comment:
drop this check.  There's nothing about this function that would suggest that users could pass None as their scheduler.

Also, if they want to pass bone-headed values in, there's a million things they could pass that are not None but which would also cause problems, so this check is like a really incomplete blacklist of bad values.

Reply:
I was thinking the same thing, especially given the type-hinting.

Reply:
Removed.